ci/docs/testrubyserver.rb

```diff
@@ -3,12 +3,12 @@
 include WEBrick
 
 config = {}
-config.update(:Port => 8080)
+config.update(:Port => 24816)
```

Comment:
is this used only for pulp3 or also  2?

Reply:
Only for Pulp 3.

Reply:
I think 8080 was only ever used by Pulp3:
https://docs.pulpproject.org/en/2.18/user-guide/installation/index.html#prerequisites